File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -0,0 +1,932 @@
+	FileStream outputStream = new FileStream("Output.pptx", FileMode.Create);
+	pptxDoc.Save(outputStream);
+	//Release all resources of the stream
+	outputStream.Dispose();
+	//Closes the Presentation instance
+	pptxDoc.Close();
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+Stream inputStream = assembly.GetManifestResourceStream("SampleBrowser.Presentation.Samples.Template.Sample.pptx");
+//Loads or open an PowerPoint Presentation
+IPresentation pptxDoc = Presentation.Open(inputStream);
+//Finds all the occurrences of a particular text in the PowerPoint presentation
+ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+foreach (ITextSelection textSelection in textSelections)
+{
+	//Gets the found text as single text part
+	ITextPart textPart = textSelection.GetAsOneTextPart();
+	//Replace the text
+	textPart.Text = "Service";
+}
+//Create new memory stream to save Presentation.
+MemoryStream stream = new MemoryStream();
+//Save Presentation in stream format.
+pptxDoc.Save(stream);
+//Close the presentation
+pptxDoc.Close();
+stream.Position = 0;
+//The operation in Save under Xamarin varies between Windows Phone, Android and iOS platforms. Please refer presentation/xamarin section for respective code samples.
+if (Device.OS == TargetPlatform.WinPhone || Device.OS == TargetPlatform.Windows)
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+### Match case
+
+You can find and replace the text by matching case also.
+
+The following code example illustrates how to find all occurences of a particular text by matching its case.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+	bool matchCase = true;
+	bool wholeWord = false;
+	//Finds all the occurrences of a particular text which matches the given casing
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", matchCase, wholeWord);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text as single text part
+		ITextPart textPart = textSelection.GetAsOneTextPart();
+		//Replace the text
+		textPart.Text = "Service";
+	}
+	//Saves the Presentation	
+	pptxDoc.Save("Output.pptx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Opens an existing presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+	Dim matchCase = True
+	Dim wholeWord = False
+	'Finds all the occurrences of a particular text which matches the given casing
```

Comment:
Finds all the occurrences of a particular text which matches the given casing

Reply:
Modified as you said.